style.css

```diff
@@ -0,0 +1,181 @@
+body{
+    color: #7069f3;
+}
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+}
+
+.hamburger{
+    position : absolute;
+    z-index: 2;
+    right: 0;
+}
+
+h1{
+    text-align: center;
+    padding: .1em 0;
+}
+
+.overflow-auto{
+    overflow-y: auto;
+    scrollbar-width: thin;      
+    scrollbar-color: #7069f3 #fff;  
+}
+.overflow-auto::-webkit-scrollbar {
+    width: 2px;               /* width of the entire scrollbar */
```

Comment:
Too small width to easily get hold of with a mouse. Some people click and hold the scrollbar thumb and drag it. They won;t be able to click this thin thumb